Changes according to review

JiHaripriya opened the pull request:
- Added alt text to the image.
- Used shorthand properties for the border, padding, and margin.
- Applied auto height property to images.
- Added proper comments in the CSS.
- Added button border properties.
- Added hover property to horizontal navigation.

JiHaripriya committed:
Changes according to review

JiHaripriya committed:
Add home page

JiHaripriya committed:
Add about and menu section

JiHaripriya committed:
Renamed update image file under outputs

JiHaripriya committed:
Add contact section and footer

JiHaripriya committed:
Added comments and spacing to html and stylesheet

Priyaqb commented:
- Include normalize before style css.
- Don't include logo along with the nav tag.
- Go through BEM class naming method.
- Remove aside and article tags used because its purpose is different.
- Try not to repeat `<h1>` tag.
- Avoid repetition of styles. Use class name instead of id.
- Don't use height for contents.
- Add :hover styles for nav, buttons and links. It can be a color change.

JiHaripriya committed:
Minor changes in HTML tags and css

JiHaripriya committed:
Added hover functionality

JiHaripriya committed:
Add changes in alignment of footer

JiHaripriya committed:
Fix alignment issues

JiHaripriya committed:
Modified font size as per suggestion

JiHaripriya committed:
CSS comments and code optimization

JiHaripriya commented:
> * Include normalize before style css.
> * Don't include logo along with the nav tag.
> * Go through BEM class naming method.
> * Remove aside and article tags used because its purpose is different.
> * Try not to repeat `<h1>` tag.
> * Avoid repetition of styles. Use class name instead of id.
> * Don't use height for contents.
> * Add :hover styles for nav, buttons and links. It can be a color change.

* Included normalize before style CSS.
* Separated logo from the `nav` tag.
* Removed aside and article tags and replaced it with `div`.
* Removed multiple `h1` tags.
* Optimized reusable CSS code.
* Removed usage of height for contents.
* Added a color change :hover styles for nav, buttons, and links.

JiHaripriya committed:
Modified footer content

JiHaripriya committed:
Added output and modified README

JiHaripriya committed:
Renamed folders

JiHaripriya committed:
Added output link to README

JiHaripriya committed:
Added missing quotes in README

JiHaripriya committed:
Changed color for hover
